apps/condo/domains/organization/access/OrganizationEmployeeSpecialization.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Generated by `createschema organization.OrganizationEmployeeSpecialization 'employee:Relationship:OrganizationEmployee:CASCADE; specialization:Relationship:TicketCategoryClassifier:CASCADE;'`
+ */
+
+const get = require('lodash/get')
+
+const { throwAuthenticationError } = require('@condo/keystone/apolloErrorFormatter')
+const { getById } = require('@condo/keystone/schema')
+const { isSoftDelete } = require('@condo/keystone/access')
+
+const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor, checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+
+async function canReadOrganizationEmployeeSpecializations ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    return {
+        employee: {
+            organization: {
+                OR: [
+                    queryOrganizationEmployeeFor(user.id),
+                    queryOrganizationEmployeeFromRelatedOrganizationFor(user.id),
+                ],
+            },
+        },
+    }
+}
+
+async function canManageOrganizationEmployeeSpecializations ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    if (operation === 'create') {
+        const employeeId = get(originalInput, ['employee', 'connect', 'id'])
+        if (!employeeId) return false
+
+        const employee = await getById('OrganizationEmployee', employeeId)
+        const organizationId = get(employee, 'organization')
+
+        return await checkOrganizationPermission(user.id, organizationId, 'canManageEmployees')
```

Comment:
it looks like we need another one access for that

Reply:
canManage<MODEL-NAME>

Reply:
you can create a virtual field inside the EmployeeRole for that if you don't need a real field and get the value from canManageEmployees

Reply:
`checkOrganizationPermission` uses `find` functions, they are not resolve virtual fields